WordPress/Classes/ViewRelated/Reader/ReaderPostCardCell.swift

```diff
@@ -232,11 +245,13 @@ private struct Constants {
         summaryLabel.backgroundColor = .listForeground
         commentActionButton.titleLabel?.backgroundColor = .listForeground
         likeActionButton.titleLabel?.backgroundColor = .listForeground
+        topicsCollectionView.backgroundColor = .listForeground
```

Comment:
Maybe move that to `TopicsCollectionView`?

Reply:
> Maybe move that to `TopicsCollectionView`?

I think that the this should be configured by the implementing view since it could change depending on where else it's being used. Thoughts?

Reply:
Makes sense! Ok!